draft-sasikumar-id-claim-169.md

```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
 
-```CWT
-   {
-      "1":"COUN",
-      "6":1665980929,
-      "8":{
-         "3":"dfd1aa976d8d4575a0fe34b96de2bfad"
-      },
-      "169":{
-         "1": "11110000324013",
-         "2": "1.0",
-         "3": "EN",
-         "4": "Peter M Jhon",
-         "5": "Peter",
-         "6": "M",
-         "7": "Jhon",
-         "8": "19880102",
-         "9": 1,
-         "10": "New City, METRO LINE, PA",
-         "11": "peter@example.com",
-         "12": "+1 234-567",
-         "13": "US",
-         "14": 2,
-         "15": "Jhon Honai",
-         "16": "03CBABDF83D068ACB5DE65B3CDF25E0036F2C546CB90378C587A076E7A759DFD27CA7872B6CDFF339AEAACA61A6023FD1D305A9B4F33CAA248CEDE38B67D7C915C59A51BB4E77D10077A625258873183F82D65F4C482503A5A01F41DEE612C3542E5370987815E592B8EA2020FD3BDDC747897DB10237EAD179E55B441BC6D8BAD07CE535129CF8D559445CC3A29D746FBF1174DE2E7C0F3439BE7DBEA4520CF88825AAE6B1F291A746AB8177C65B2A459DD19BD32C0C3070004B85C1D63034707CC690AB0BA023350C8337FC6894061EB8A714A8F22FE2365E7A904C72DEC9746ABEA1A3296ECACD1A40450794EDCD2B34844E7C19EB7FB1A4AF3B05C3B374BD2941603F72D3F9A62EAB9A2FDAEEEEC8EE6E350F8A1863C0A0AB1B4058D154559A1CD5133EFCF682ABC339960819C9427889D60380B635A7D21D017974BBA57798490F668ADD86DA58125D9C4C1202CA1308F7734E43E8F77CEB0AF968A8F8B88849F9B98B26620399470ED057E7931DED82876DCA896A30D0031A8CBD7B9EDFDF16C15C6853F4F8D9EEC09317C84EDAE4B349FE54D23D8EC7DC9BB9F69FD7B7B23383B64F22E25F",
-         "17": 2,
-         "18": [1,2],
-   }
+```yaml
+1: COUN # iss
+6: 1665980929 # iat
+8: # cnf
+  3: dfd1aa976d8d4575a0fe34b96de2bfad # kid
+169: # identity-data
+  1: "11110000324013" # ID
+  2: "1.0" # Version
+  3: EN # Language
+  4: Peter M Jhon # Full name
+  5: Peter # First name
+  6: M # Middle name
+  7: Jhon # Last name
+  8: "19880102" # Date of birth
+  9: 1 # Gender: Male
+  10: New City, METRO LINE, PA # Address
+  11: peter@example.com # Email ID
+  12: "+1 234-567" # Phone number
+  13: US # Nationality
+  14: 2 # Marital status: Married
+  15: Jhon Honai # Guardian
+  16: 03CBABDF83D068ACB5DE65B3CDF25E0036F2C546CB90378C587A076E7A759DFD27CA7872B6CDFF339AEAACA61A6023FD1D305A9B4F33CAA248CEDE38B67D7C915C59A51BB4E77D10077A625258873183F82D65F4C482503A5A01F41DEE612C3542E5370987815E592B8EA2020FD3BDDC747897DB10237EAD179E55B441BC6D8BAD07CE535129CF8D559445CC3A29D746FBF1174DE2E7C0F3439BE7DBEA4520CF88825AAE6B1F291A746AB8177C65B2A459DD19BD32C0C3070004B85C1D63034707CC690AB0BA023350C8337FC6894061EB8A714A8F22FE2365E7A904C72DEC9746ABEA1A3296ECACD1A40450794EDCD2B34844E7C19EB7FB1A4AF3B05C3B374BD2941603F72D3F9A62EAB9A2FDAEEEEC8EE6E350F8A1863C0A0AB1B4058D154559A1CD5133EFCF682ABC339960819C9427889D60380B635A7D21D017974BBA57798490F668ADD86DA58125D9C4C1202CA1308F7734E43E8F77CEB0AF968A8F8B88849F9B98B26620399470ED057E7931DED82876DCA896A30D0031A8CBD7B9EDFDF16C15C6853F4F8D9EEC09317C84EDAE4B349FE54D23D8EC7DC9BB9F69FD7B7B23383B64F22E25F # Binary image
+  17: 2 # Binary image format: JPEG
+  18: [1, 2] # Best quality fingers
+  50: # Right Thumb Biometrics
```

Comment:
This should be a [Biometrics] right. The example representations looks different. Can we change that?

Reply:
In yaml format, the array are allowed to represent using braces [] or with new line and tab separator. As we have more attributes in attribute so specified with new line.